src/configurations/sources/shopify/ui-config.json

```diff
@@ -5,10 +5,10 @@
       "fields": [
         {
           "type": "checkbox",
-          "label": "Disable client side identifier",
+          "label": "Disable client side identifier in the Legacy Shopify Tracker App",
           "value": "disableClientSideIdentifier",
           "default": false,
-          "footerNote": "Disables client side identifier in shopify tracker"
+          "footerNote": "Only applicable for the Legacy Shopify Tracker App"
```

Comment:
can we mention the app name that comes up in shopify store

Reply:
> Only applicable for RudderStack, the legacy Shopify Tracker App


updated the footernote